modules/apps/batch-planner/batch-planner-web/test/setup.js

```diff
@@ -25,23 +25,24 @@ window.Liferay.Language = {
 };
 
 window.Liferay.ThemeDisplay = {
-	getBCP47LanguageId: () => 'en-US',
+	...window.Liferay.ThemeDisplay,
 	getPathThemeImages: () => '/assets',
 };
 
-window.Liferay.detach = (name, fn) => {
-	window.removeEventListener(name, fn);
-};
 window.Liferay.fire = (name, payload) => {
```

Comment:
Do we not provide a default mock for fire and on? Those seem like common utils that we could copy this implementation for our global mocks

Reply:
We don't, and they are not 1:1 implementations, if I move these to our global mocks, the tests that relied on them start failing. I'll leave these for the end, but that's why I didn't move these ones before, i couldn't figure out how to move them to global mocks without tests starting to fail